### DISCUSSION TITLE
Cluster

### DISCUSSION THREAD

**Discussion by paul-gauci** [jodal/pykka#178] [Ideas]
Will Pykka ever support a cluster of nodes?

**Comment by jodal** [comment]
No, I don't think that is realistic to expect Pykka to ever support that.

Pykka has been quite stable doing what it currently does well for a number of years, and I have no immediate plans to extend the set of features.

**Reply by paul-gauci** [reply]
I understand...pity cause I like it very much, unfortunately I cannot do it myself due to my limited knowledge
